<?xml version="1.0"?>
 <launch>

  <rosparam file="$(find movo_hardware_interface)/config/hardware.yaml" command="load"/>

  <rosparam file="$(find movo_hardware_interface)/config/right_arm_7dof_controller.yaml" command="load"/>
  <rosparam file="$(find movo_hardware_interface)/config/right_gripper_controller.yaml" command="load"/>
  <rosparam file="$(find movo_hardware_interface)/config/left_arm_7dof_controller.yaml" command="load"/>
  <rosparam file="$(find movo_hardware_interface)/config/left_gripper_controller.yaml" command="load"/>
  <rosparam file="$(find movo_hardware_interface)/config/torso_controller.yaml" command="load"/>
  <rosparam file="$(find movo_hardware_interface)/config/head_controller.yaml" command="load"/>

  <rosparam file="$(find movo_hardware_interface)/config/joint_state_controller.yaml" command="load"/>
  <rosparam file ="$(find movo_hardware_interface)/config/controllers.yaml" command="load"/>
  
  <node name="movo_hardware_interface" pkg="movo_hardware_interface" type="movo_hardware_interface_node" output="screen"/>
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
		args="movo/right_arm_controller
		      movo/left_arm_controller 
		      movo/head_controller
		      movo/torso_controller
		      movo/left_gripper_controller
 		      movo/right_gripper_controller
                      joint_state_controller
		"/>

</launch>
